agents/check_mk_agent.openbsd

```diff
@@ -217,6 +217,11 @@ section_misc_sections() {
             # Speed
             SP=$(echo "$IO" | grep -E "media:.*base" | cut -d\( -f2 | cut -db -f1)
             if [ "$SP" ]; then
+                case "$SP" in
+                        # Detect 10Gbps interfaces
+                        *G*) SP=10000
```

Comment:
This would break on 20G or 40G NICs. We should try to convert the value correctly instead of just guessing this is a 10G NIC.